gateway/auth_manager.go

```diff
@@ -281,7 +286,21 @@ func (b *DefaultSessionManager) SessionDetail(keyName string, hashed bool) (user
 	if hashed {
 		jsonKeyVal, err = b.store.GetRawKey(b.store.GetKeyPrefix() + keyName)
 	} else {
-		jsonKeyVal, err = b.store.GetKey(keyName)
+		// try to get legacy and new format key at once
```

Comment:
We can actually detect if key is "imported". Smt like:
`If storage.TokenOrg(key) == ""`

Reply:
Mget needed only for custom keys